deploy-board/deploy_board/templates/environs/env_landing.html

```diff
@@ -165,11 +165,9 @@ <h4 class="modal-title">Restart Confirm</h4>
         <h4 class="modal-title">Pause Confirm</h4>
       </div>
       <div class="modal-body">
-        <p>Are you sure to pause the current deploy?</p>
-        <div class="alert alert-danger" role="alert">
-            <strong>WARNING</strong>:
-            Pausing a deploy is not recommended and should be used with extreme caution. Please note that environment will not be deployed properly on a new host(s) added manually or through ASG autoscaling, while deployment is in PAUSED state.
-        </div>
+        <p>Are you sure to pause the current deploy?
+           Deployment will be paused on existing instances of a cluster but will continue on any new hosts added to a cluster. 
```

Comment:
can you add more info such as:  please note that pausing a deployment may lead to your cluster serving 2 different builds (existing deploy and new deploy).  It is recommended to only leave the deployment in "pause" state temporarily.